packages/contracts/test/teller-v2.spec.ts

```diff
@@ -361,7 +371,7 @@ describe('TellerV2', () => {
         .emit(tellerV2, 'AcceptedBid')
         .withArgs(bidId, lender.address)
 
-      const stakeholder = await tellerV2.getLoanLender(bidId)
+      const stakeholder = await lenderManager.getActiveLoanLender(bidId)
```

Comment:
no longer have this function - please revert and make sure the JS tests are still working